source/DistanceMap.cpp

```diff
@@ -41,7 +41,7 @@ DistanceMap::DistanceMap(const PlayerInfo &player, const System *center)
 {
 	if(!player.Flagship())
 		return;
-	
+
```

Comment:
@EndrosG If I make another commit which fixes the spaces, then it will also be huge. Should I close this and make a new clean PR, or should I do it anyways?